components/BakeryMenu.js

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+import Image from 'next/image';
+
+export default function BakeryMenu({ image, name, ingredients, weight, cost }) {
+  return (
+    <MenuCard>
+      <TextWrapper>
+        <h4>{name}</h4>
+        <IngredientWrapper>{ingredients}</IngredientWrapper>
+        <Text>{weight}</Text>
+        <Text className="cost">{cost}</Text>
+      </TextWrapper>
+      <Image src={image} layout="responsive" width={60} height={60} />
```

Comment:
```suggestion
      <Image src={image} layout="fixed" width={60} height={60} />
```

I don't know if this will make a difference when rendering, but i think the width and height are only applied with layout="fixed". In case it has the same effect, just ignore me 😄

Reply:
Exactly what I was looking for, thank you!